pkg/process/procutil/data_scrubber_windows_test.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package procutil
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStripArguments(t *testing.T) {
+
+	cases := []struct {
+		cmdline       []string
+		noArgsCmdline []string
+	}{
+	// Main cases samples 
+		{[]string{"agent", "-password", "1234"}, []string{"agent"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.com"}, []string{"C:\\Program Files\\Datadog\\agent.com"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.exe check process"}, []string{"C:\\Program Files\\Datadog\\agent.exe"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.bat", "check", "process"}, []string{"C:\\Program Files\\Datadog\\agent.bat"}},
+		{[]string{"\"C:\\Program Files\\Datadog\\agent.cmd\" check process"}, []string{"C:\\Program Files\\Datadog\\agent.cmd"}},
+	
+	// String matching extension structure
+			{[]string{"C:\\Program File\\Datexedog\\agent.exe check process"}, []string{"C:\\Program File\\Datexedog\\agent.exe"}},
+
+	// Mixed Variables	
+		{[]string{"C:\\Program Files\\agent.vbs check process"}, []string{"C:\\Program Files\\agent.vbs"}},
+		{[]string{"\"C:\\Program Files\\Datadog\\agent.vbe\" check process"}, []string{"C:\\Program Files\\Datadog\\agent.vbe"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.js", "check", "process"}, []string{"C:\\Program Files\\Datadog\\agent.js"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.jse check process"}, []string{"C:\\Program Files\\Datadog\\agent.jse"}},
+		{[]string{"\"C:\\Program Files\\Datadog\\agent.wsf\" check process"}, []string{"C:\\Program Files\\Datadog\\agent.wsf"}},
+		{[]string{"C:\\Program Files\\Datadog\\agent.wsh check process"}, []string{"C:\\Program Files\\Datadog\\agent.wsh"}},
+		{[]string{"\"C:\\Program Files\\Datadog\\agent.psc1\" check process"}, []string{"C:\\Program Files\\Datadog\\agent.psc1"}},
+		{[]string{"\"C:\\Program Files\\agent\" check process"}, []string{"C:\\Program Files\\agent"}},
+		
+	}
+	
+	scrubber := setupDataScrubber(t)
+	scrubber.StripAllArguments = true
+
+	for i := range cases {
```

Comment:
[minor] Please write the table tests using the subtest pattern: https://go.dev/blog/subtests. Example from my [PR](https://github.com/DataDog/datadog-agent/pull/22450/files#diff-3e43a7498175fffb646965d42b06e15255c702c8441b23c13f59986221883e31)

Reply:
This also applies for the other tests you have.

Reply:
By "subtest pattern", what that means is to use `t.Run()` ([docs here](https://pkg.go.dev/testing#T.Run), [example here](https://go.dev/blog/subtests#:~:text=Table%2Ddriven%20tests%20using%20subtests)). You would do it like this (and please make this change to your other tests)

```
for i := range cases {
  t.Run(strconv.Itoa(i), func(t *testing.T) {
    ... stripArguments() ...
    ... assert.Equal() ...
  }
}
```